geth/transactions/txqueue_manager.go

```diff
@@ -29,6 +29,7 @@ type Manager struct {
 	accountManager common.AccountManager
 	txQueue        *queue.TxQueue
 	ethTxClient    EthTransactor
+	txSender       ethereum.TransactionSender
```

Comment:
i don't like this change, it is possible to implement tests without this

Reply:
Me neither, but it boils down to separating what `EthTransactor` consists of if this is a valid way to test transaction values. I know it looks like duplication.